hack/generate-yaml.sh

```diff
@@ -121,7 +121,7 @@ ${HELM_BIN_DIR}/helm template ${VK_ROOT}/installer/helm/chart/volcano --namespac
       >> ${DEPLOYMENT_FILE}
 
 ${HELM_BIN_DIR}/helm template ${VK_ROOT}/installer/helm/chart/volcano --namespace volcano-monitoring \
-      --name-template volcano --set basic.image_tag_version=${VOLCANO_IMAGE_TAG} \
+      --name-template volcano --set basic.image_tag_version=${VOLCANO_IMAGE_TAG} --set custom.metrics_enable=true \
```

Comment:
Maybe we should disable it by default?

Reply:
actually this will only render `volcanno-monitoring-latest.yaml`, which I think not in real use.